src/ucp/dt/dt.c

```diff
@@ -44,3 +45,51 @@ size_t ucp_dt_pack(ucp_datatype_t datatype, void *dest, const void *src,
     state->offset += result_len;
     return result_len;
 }
+
+ucs_status_t
```

Comment:
is it beneficial to have it non-inlined?

Reply:
the problem is in cross dependency of headers. The old implementation was buggy (not using `UCP_RECV_DESC_FLAG_LAST`). See last arg, it's changed from `int last` to `unsigned flags`

Reply:
need to solve it another way, this functions should be inlined since it's fast-path